packages/sources/finnhub-secondary/src/config/includes.json

```diff
@@ -0,0 +1,189 @@
+[
+  {
+    "from": "DKK",
+    "to": "USD",
+    "includes": [
+      {
+        "from": "USD",
+        "to": "DKK",
+        "inverse": true
+      }
+    ]
+  },
+  {
+    "from": "JPY",
+    "to": "USD",
+    "includes": [
+      {
+        "from": "USD",
+        "to": "JPY",
+        "inverse": true
+      }
+    ]
+  },
+  {
+    "from": "TRY",
+    "to": "USD",
+    "includes": [
+      {
+        "from": "USD",
+        "to": "TRY",
+        "inverse": true
+      }
+    ]
+  },
+  {
+    "from": "MXN",
+    "to": "USD",
+    "includes": [
+      {
+        "from": "USD",
+        "to": "MXN",
+        "inverse": true
+      }
+    ]
+  },
+  {
+    "from": "ZAR",
+    "to": "USD",
+    "includes": [
+      {
+        "from": "USD",
+        "to": "ZAR",
+        "inverse": true
+      }
+    ]
+  },
+  {
+    "from": "HKD",
+    "to": "USD",
+    "includes": [
+      {
+        "from": "USD",
+        "to": "HKD",
+        "inverse": true
+      }
+    ]
+  },
+  {
+    "from": "NOK",
+    "to": "USD",
+    "includes": [
+      {
+        "from": "USD",
+        "to": "NOK",
+        "inverse": true
+      }
+    ]
+  },
+  {
+    "from": "HUF",
+    "to": "USD",
+    "includes": [
+      {
+        "from": "USD",
+        "to": "HUF",
+        "inverse": true
+      }
+    ]
+  },
+  {
+    "from": "CNH",
+    "to": "USD",
+    "includes": [
+      {
+        "from": "USD",
+        "to": "CNH",
+        "inverse": true
+      }
+    ]
+  },
+  {
+    "from": "SEK",
+    "to": "USD",
+    "includes": [
+      {
+        "from": "USD",
+        "to": "SEK",
+        "inverse": true
+      }
+    ]
+  },
+  {
+    "from": "PLN",
+    "to": "USD",
+    "includes": [
+      {
+        "from": "USD",
+        "to": "PLN",
+        "inverse": true
+      }
+    ]
+  },
+  {
+    "from": "SGD",
+    "to": "USD",
+    "includes": [
+      {
+        "from": "USD",
+        "to": "SGD",
+        "inverse": true
+      }
+    ]
+  },
+  {
+    "from": "CZK",
+    "to": "USD",
+    "includes": [
+      {
+        "from": "USD",
+        "to": "CZK",
+        "inverse": true
+      }
+    ]
+  },
+  {
+    "from": "WTI",
+    "to": "USD",
+    "includes": [
+      {
+        "from": "WTICO",
+        "to": "USD",
+        "inverse": true
```

Comment:
Comparing with the removed `overrides.json`, should this specific mapping really be `inverse`-ed?

Reply:
others looked good to me, but worth a second pass to make sure I didn't miss anything either!

Reply:
Glad you spotted it - while checking I noticed there are non-FX overrides that we need to keep, that I'd missed earlier.